src/makeRobotAccountant.js

```diff
@@ -6,7 +6,24 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = -1;
+
+  function getSum(number1) {
+    counter++;
+
+    function getSum2(number2) {
+      switch (true) {
+        case (counter < 3 || (counter % 2 === 0 && counter > 3)):
+          return number1 + number2;
+        case (counter % 2 !== 0 && counter >= 3):
+          return 'Bzzz... Error!';
+      }
```

Comment:
And let's simplify your condition to only one

Reply:
You need to check only if calls count > 3 && calls count % 2 === 0
So in this case result should be 'Bzzz... Error!' 
I other cases sum of numbers